src/Echidna.Tests/Echidna.Tests.csproj

```diff
@@ -22,11 +22,11 @@
     <PackageReference Include="NUnit3TestAdapter" Version="4.0.0" />
     <PackageReference Include="coverlet.collector" Version="3.1.0" />
 
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="4.0.0" />
-    <PackageReference Include="System.Data.SqlClient" Version="4.8.3" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="5.1.5" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.6" />
     <PackageReference Include="Npgsql" Version="6.0.2" />
     <PackageReference Include="MySqlConnector" Version="2.1.2" />
-    <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="3.21.50" />
+    <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="3.21.130" />
```

Comment:
https://www.nuget.org/packages/Oracle.ManagedDataAccess.Core#versions-body-tab

This is the latest version.